ios/REANodesManager.m

```diff
@@ -174,12 +196,36 @@ - (void)performOperations
   if (_operationsInBatch.count != 0) {
     NSMutableArray<REANativeAnimationOp> *copiedOperationsQueue = _operationsInBatch;
     _operationsInBatch = [NSMutableArray new];
+    
+    __weak typeof(self) weakSelf = self;
+    dispatch_semaphore_t semaphore = dispatch_semaphore_create(0);
     RCTExecuteOnUIManagerQueue(^{
+      NSMutableArray *pendingUIBlocks = [self.uiManager valueForKey:@"_pendingUIBlocks"];
+      bool canPerformLayout = (pendingUIBlocks == nil) || ([pendingUIBlocks count] == 0);
```

Comment:
`[_pendingUIBlocks count]` will return 0 when `pendingUIBlocks` is `nil` so there is no need to have these two checks